packages/api/tests/models/test-collections-model.js

```diff
@@ -100,13 +87,21 @@ test('Collection.delete() throws an exception if the collection has associated r
 test('Collection.delete() deletes a collection', async (t) => {
   const name = randomString();
   const version = randomString();
+  const item = fakeCollectionFactory({ name, version });
+  const { dataType } = item;
+  const collectionConfigStore = collectionsModel.collectionConfigStore;
+  const collectionId = constructCollectionId(dataType, version);
```

Comment:
So I think `constructCollectionId` expects name/version, or at least those parameter names? But the `collectionConfigStore` class internally treats the collection ID as the data type/version, which is why you're constructing it that way here. 

In other words, there's nothing wrong here, just wanted to make sure I understand.